cgiutil.h

```diff
@@ -70,6 +70,12 @@ typedef struct {
   char *postrequest;
 
   char *httpcookiedata;
+
+  // todo, SWIG?
```

Comment:
Wrap these in a ifndef statement for now?

#ifndef SWIG
  char *path_info;
  int api_path_length;
  char **api_path;
#endif

Reply:
I think you're right. Param names and values are also hidden...